pages/api/admins.ts

```diff
@@ -36,5 +40,20 @@ export default withApiAuth(
         return res.status(500).json({ message: "Internal server error" });
       }
     }),
+    DELETE: withApiValidation({ body: deleteAdminSchema }, async (req, res) => {
+      const { ids } = req.body;
+
+      try {
+        ids.forEach(async (adminId) => {
+          await prisma.admin.delete({
+            where: { id: adminId },
+          });
+        });
```

Comment:
This should be wrapped into one database transaction https://www.prisma.io/docs/concepts/components/prisma-client/transactions

```suggestion
        await prisma.$transaction(ids.map((adminId) => prisma.admin.delete({
            where: { id: adminId },
          });
        ));
```